x-pack/test/detection_engine_api_integration/security_and_spaces/tests/import_rules.ts

```diff
@@ -95,8 +97,136 @@ export default ({ getService }: FtrProviderContext): void => {
   const supertest = getService('supertest');
   const log = getService('log');
   const esArchiver = getService('esArchiver');
+  const supertestWithoutAuth = getService('supertestWithoutAuth');
 
   describe('import_rules', () => {
+    describe('importing rules with different roles', () => {
+      before(async () => {
+        await createUserAndRole(getService, ROLES.hunter_no_actions);
+        await createUserAndRole(getService, ROLES.hunter);
+      });
+      after(async () => {
+        await deleteUserAndRole(getService, ROLES.hunter_no_actions);
+        await deleteUserAndRole(getService, ROLES.hunter);
+      });
+      beforeEach(async () => {
+        await createSignalsIndex(supertest, log);
+      });
+
+      afterEach(async () => {
+        await deleteSignalsIndex(supertest, log);
+        await deleteAllAlerts(supertest, log);
+      });
+      it('should successfully import rules without actions when user has no actions privileges', async () => {
```

Comment:
Per zoom discussion: add logic to skip connector check if no actions are being imported so that user does not receive this error message.

Thanks so much for quickly getting to this!

Reply:
resolved here: [`c1aa575` (#126203)](https://github.com/elastic/kibana/pull/126203/commits/c1aa5752cc3a93491e60a88e94909633395552bd)